client/documents/list/index.tsx

```diff
@@ -0,0 +1,176 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { dateI18n } from '@wordpress/date';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+import { TableCard, TableCardColumn } from '@woocommerce/components';
+import { onQueryChange, getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { displayType } from 'documents/strings';
+import { Document, useDocuments, useDocumentsSummary } from 'data/index';
+import './style.scss';
+import DocumentsFilters from '../filters';
+import Page from '../../components/page';
+
+interface Column extends TableCardColumn {
+	key: 'date' | 'type' | 'description' | 'download';
+	visible?: boolean;
+	cellClassName?: string;
+}
+
+const getColumns = (): Column[] =>
+	[
+		{
+			key: 'date',
+			label: __( 'Date', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Date and time', 'woocommerce-payments' ),
+			required: true,
+			isLeftAligned: true,
+			defaultOrder: 'desc',
+			cellClassName: 'date',
+			isSortable: true,
+			defaultSort: true,
+		},
+		{
+			key: 'type',
+			label: __( 'Type', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Type', 'woocommerce-payments' ),
+			required: true,
+			isLeftAligned: true,
+		},
+		{
+			key: 'description',
+			label: __( 'Description', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Description', 'woocommerce-payments' ),
+		},
+		{
+			key: 'download',
+			label: '',
+			screenReaderLabel: __( 'Download', 'woocommerce-payments' ),
+			isLeftAligned: false,
+			isNumeric: true,
+		},
+	].filter( Boolean ) as Column[]; // We explicitly define the type because TypeScript can't infer the type post-filtering.
+
+const getDocumentDescription = ( document: Document ) => {
+	switch ( document.type ) {
+		case 'test_document':
+			if ( document.period_from && document.period_to ) {
+				return sprintf(
+					__(
+						'This is a test document for %s to %s',
+						'woocommerce-payments'
+					),
+					dateI18n(
+						'M j, Y',
+						moment.utc( document.period_from ).local().toISOString()
+					),
+					dateI18n(
+						'M j, Y',
+						moment.utc( document.period_to ).local().toISOString()
```

Comment:
Should we display these dates in the server's timezone (UTC)? I added a report from 02-01 to 02-28, but it shows up as from 01-31 to 02-28 because of my timezone being UTC-3.

Reply:
Per our discussion, I've updated the dates to show in the UTC timezone in https://github.com/Automattic/woocommerce-payments/pull/4000/commits/04bfa6ae854ccf8c8cc5621085a85418f094735a

This is pending us getting further information on whether documents should be reflected with local or UTC time.

Reply:
Awesome! Should we also remove the `.local()` call? It doesn't seem to be working properly, but it could cause the dates to be in local time instead of UTC.

Reply:
Good catch with this, I had removed it while we were talking about it, and didn't realize I added it back. Removed in https://github.com/Automattic/woocommerce-payments/pull/4000/commits/8201ac01e74233e33657422332207677b0bba290